.github/workflows/django.yml

```
@@ -0,0 +1,29 @@
+name: django-testsuite
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches:
      - master
  pull_request
```

This is what we do elsewhere, otherwise you get double builds on PRs, we really just need on PRs and on merge to `master`.